File: src/gestorAplicacion/EstimuloEstudianteInterfaz.java
package gestorAplicacion;

import java.util.ArrayList;

public abstract interface EstimuloEstudianteInterfaz {
  public abstract ArrayList<String> obtenerCriterios();
  public abstract ArrayList<Estudiante> obtenerAplicantes();
  public abstract boolean verificarRequisitos(Estudiante estudiante);
}

File: PYTHON/src/gestorAplicacion/clasesExtra/EstimuloEstudianteInterfaz.py
from abc import ABC, abstractmethod

class EstimuloEstudianteInterfaz(ABC):
  @abstractmethod
  def obtenerCriterios(self):
    pass

  @abstractmethod
  def obtenerAplicantes(self):
    pass

  @abstractmethod
  def verificarRequisitos(self, estudiante):
    pass
